openjdk/mmtkBarrierSetAssembler_x86.cpp

```diff
@@ -111,7 +113,8 @@ void MMTkBarrierSetAssembler::eden_allocate(MacroAssembler* masm, Register threa
     // lab.cursor = end
     __ movptr(cursor, end);
 
-#ifdef MMTK_ENABLE_GLOBAL_ALLOC_BIT
+// #ifdef MMTK_ENABLE_GLOBAL_ALLOC_BIT
```

Comment:
With this change, for plans with alloc bit on (e.g. semispace with alloc bit turned on), you are not generating alloc bit code for them. This seems incorrect. But I have no idea how the tests passed. Any idea? @YiluoWei

Reply:
I think before testing the global alloca bit, the CI tests already failed.

Probably we need some changes in mmtk-core so that when global alloc bit is enabled, the allocator tag (`TAG_BUMP_POINTER`) in other plans becomes `TAG_BUMP_POINTER_ALLOC_BIT`.

Reply:
Ah I see. I thought the tests were in the test-assertions. Thanks Yiluo.